duckduckgo/android-search-and-stories
# removed browser integration prompt

Reviewer: @subsymbolic 
Asana: https://app.asana.com/0/19626896268257/404579766834639/f
CC:

**Description**:

Removes the prompts to integrate with Chrome and Firefox and the feed header.  Also removed a fair amount of other "unused" code.

**Steps to test this PR**:
1. Launch the app.  You should immediately see the feed with no integration prompt popup or dismissible panel at the top of the feed.


###### Reviewer Checklist:


###### PR DRI Checklist:
